NOTICE.txt

```diff
@@ -1,314 +1,765 @@
+	Homepage:	https://github.com/kaazing/gateway.service.update.check.management.git
 
-This product depends on BIO Transport 1.0
+This product depends on Keystore and Truststore used for testing TLS 1.0.0.24
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	https://github.com/kaazing/gateway.test.ca.git
+
+This product depends on Transport Core develop-SNAPSHOT
+
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	https://github.com/kaazing/gateway.transport.git
+
+This product depends on BIO Transport develop-SNAPSHOT
+
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
 	Homepage:	https://github.com/kaazing/gateway.transport.bio.git
 
-This product depends on Http Transport 1.0
+This product depends on Http Transport develop-SNAPSHOT
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
 	Homepage:	https://github.com/kaazing/gateway.transport.http.git
 
-This product depends on NIO Transport 1.0
+This product depends on NIO Transport develop-SNAPSHOT
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
 	Homepage:	https://github.com/kaazing/gateway.transport.nio.git
 
-This product depends on Pipe Transport 1.0
+This product depends on Pipe Transport develop-SNAPSHOT
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
 	Homepage:	https://github.com/kaazing/gateway.transport.pipe.git
 
-This product depends on Sse Transport 1.0
+This product depends on Sse Transport develop-SNAPSHOT
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
 	Homepage:	https://github.com/kaazing/gateway.transport.sse.git
 
-This product depends on SSL Transport 1.0
+This product depends on SSL Transport develop-SNAPSHOT
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
 	Homepage:	https://github.com/kaazing/gateway.transport.ssl.git
 
-This product depends on WS Transport 1.0
+This product depends on WS Transport develop-SNAPSHOT
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
```

Comment:
Can we not include kaazing stuff by chance, as this version would change everytime?